python/pyarrow/tests/test_io.py

```diff
@@ -1106,11 +1119,13 @@ def _check_native_file_reader(FACTORY, sample_data,
     assert f.tell() == ex_length
 
 
+@pytest.mark.numpy
 def test_memory_map_reader(sample_disk_data):
```

Comment:
Another to do item for a follow-up to rewrite `sample_disk_data` to not use numpy